### PR TITLE
MESH-861: More config

### PR DESCRIPTION
**Pull request by adamdossa**
This also renames general_tm => compliance_manager.

### CODE REVIEW

File: pallets/identity/src/lib.rs [hunk 1]
```diff
@@ -627,25 +627,6 @@ decl_module! {
...
-        pub fn get_my_did(origin) -> DispatchResult {
-            let sender_key = AccountKey::try_from(ensure_signed(origin)?.encode())?;
-            let did = Context::current_identity_or::<Self>(&sender_key)?;
-
-            Self::deposit_event(RawEvent::DidQuery(sender_key, did));
-            Ok(())
-        }
-
-        #[weight = SimpleDispatchInfo::FixedNormal(200_000)]
-        pub fn get_cdd_of(_origin, of: T::AccountId) -> DispatchResult {
```
Comment (maxsam4) [inline pallets/identity/src/lib.rs:640]: These are super handy during testing, can we please keep them for the testnet? :)

Comment (adamdossa) [reply pallets/identity/src/lib.rs:640]: What do you use them for? My thinking is that we should just be able to use an RPC function instead.

Comment (adamdossa) [reply pallets/identity/src/lib.rs:640]: I'll add them back in though if needed - doesn't matter much either way - was just tidying up ;-)

Comment (maxsam4) [reply pallets/identity/src/lib.rs:640]: It's easier to call these functions from the UI than to make an RPC call :)

**Comment by adamdossa** [comment]
> You renamed something and did not merge the changes immediately?
> 
> Enjoy your gazillion merge conflicts.
> 
> Yours sincerely,
> git

Yep - that felt like a couple of hours of my life I won't get back anytime soon. Aiming to merge asap to avoid more conflicts ;-).

**Pull request merged**